Sources/TuistGenerator/Linter/TargetActionLinter.swift

```diff
@@ -13,6 +13,8 @@ protocol TargetActionLinting {
 
 class TargetActionLinter: TargetActionLinting {
     func lint(_ action: TargetAction) -> [LintingIssue] {
+        guard !action.isEmbeddedScript else { return [] }
```

Comment:
is this actually necessary? The below checks will be skipped either way. You could then get rid of `isEmbeddedScript` property altogether

Reply:
Good call!